### PR TITLE
Base class RawOffer in scraper.py

### PR DESCRIPTION
**Pull request by TrueMilli**
I thought this would help for categorizing offers but it doesn't.
I think it will still be helpful to harmonize things.

There are still specific RawOffers for several services but those could be harmonized in the future as well.

### CODE REVIEW

File: app/scraper/loot/steam_loot.py [hunk 1]
```diff
@@ -13,6 +12,7 @@
 from app.scraper.info.steam import skip_age_verification
 from app.scraper.info.utils import clean_loot_title
 from app.scraper.loot.scraper import Scraper
+from app.scraper.loot.steam_games import SteamRawOffer
```
Comment (eikowagenknecht) [inline app/scraper/loot/steam_loot.py:15]: The scrapers are meant to be independent of each other, please don't cross reference them

**Review by eikowagenknecht — CHANGES_REQUESTED** [review]
See inline comments

File: app/scraper/loot/scraper.py [hunk 2]
```diff
@@ -32,6 +40,10 @@ def get_duration() -> OfferDuration:
     def get_max_wait_seconds() -> int:
         return MAX_WAIT_SECONDS
 
+    @staticmethod
+    def categorize(raw_offers: list[RawOffer]) -> list[RawOffer]:
+        return raw_offers  # ToDo: Implement this method
```
Comment (TrueMilli) [inline app/scraper/loot/scraper.py:45]:
```suggestion
```

Comment (TrueMilli) [inline app/scraper/loot/scraper.py:45]:
```suggestion
```

**Review by eikowagenknecht — APPROVED** [review]
lgtm

**Pull request merged**